setup.py

```diff
@@ -500,7 +500,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'jira',
     'jsondiff',
     'mongomock',
-    'moto~=2.0',
+    'moto~=2.0.10',
```

Comment:
```suggestion
    'moto~=2.0,>=2.0.10',
```

`~=2.0.10`  is the same as `>=2.0.10,<2.1` which is more restrictive than we want I think.

Reply:
Or `mono>=2.0.10,<3` which may be straightforward to understand.

Reply:
Included in the next revision.